views/js/list-comments.js

```diff
@@ -35,6 +35,10 @@ jQuery(document).ready(function () {
   emptyProductComment.hide();
   $('.grade-stars').rating();
 
+  prestashop.on('updatedProduct', function() {
+    $('.product-comments-additional-info .grade-stars').rating();
```

Comment:
```suggestion
    $('.grade-stars').rating();
```
Why not refreshing all grade-stars items?

Reply:
@PierreRambaud 

Actually, method rating() is not checking if element has attribute `data-grade`. In `views/templates/hook/product-comment-item-prototype.tpl` (product comments list in product footer) and `views/templates/hook/product-list-reviews.tpl` (products listing, product accessories for example) we are not setting `data-grade` on an element, so after calling rating() on it (which is unnecessary) it will be set to 1 according to this code in file `views/js/jquery.rating.plugin.js`. 

```js 

var componentOptions = jQuery.extend({
      grade: null,
      input: null,
      min: 1,
      max: 5,
      starWidth: 20
    }, options);

    const minValue = Math.min(componentOptions.min, componentOptions.max);
    const maxValue = Math.max(componentOptions.min, componentOptions.max);
    const ratingValue = Math.min(Math.max(minValue, componentOptions.grade), maxValue);

```